diff --git a/recompui/src/config/ui_config_tab_graphics.cpp b/recompui/src/config/ui_config_tab_graphics.cpp
index 4fdf81f..a8d574a 100644
--- a/recompui/src/config/ui_config_tab_graphics.cpp
+++ b/recompui/src/config/ui_config_tab_graphics.cpp
@@ -149,6 +149,8 @@ namespace recompui {
         static void determine_downsampling_display(ultramodern::renderer::Resolution res_option, OptionChangeContext change_context) {
             DownsamplingOption ds_opt = get_graphics_enum_value<DownsamplingOption>(graphics::options::ds_option, change_context);
             get_graphics_config().update_option_enum_details(graphics::options::ds_option, get_downsampling_details(res_option, ds_opt));
+            // Rush 2 recomp: downsampling is only offered at Original resolution.
+            get_graphics_config().update_option_hidden(graphics::options::ds_option, res_option != ultramodern::renderer::Resolution::Original);
         }
 
         static std::string get_framerate_text(uint32_t refresh_rate) {
